docker/docker-compose.yml

```diff
@@ -478,7 +478,7 @@ services:
       context: stacks-api
       args:
         GIT_URI: "https://github.com/hirosystems/stacks-blockchain-api.git"
-        GIT_BRANCH: "v8.0.3"
```

Comment:
These updates are due to that when i cleaned by environment and these got rebuilt, the stacks-api started crashing with an event they didn't know about (`/new_mempool_tx` or something like that).  Bumping the version fixed it.